From ad5fe65d061ef84921d6d455005984fd4e5a0b64 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Tue, 6 Apr 2021 15:44:12 -0500
Subject: [PATCH 23/41] HSD #22012814951-1: intel_fcs: don't report error with
 rejected service

commit 499e23bbf3db034138e246d27d64be9a54507dde from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.114-lts

Change to not report mailbox error when service request was rejected.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/crypto/intel_fcs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/intel_fcs.c b/drivers/crypto/intel_fcs.c
index b13746df2ff5..f203113c13a5 100644
--- a/drivers/crypto/intel_fcs.c
+++ b/drivers/crypto/intel_fcs.c
@@ -68,7 +68,7 @@ static void fcs_data_callback(struct stratix10_svc_client *client,
 		priv->size = (data->kaddr3) ?
 			*((unsigned int *)data->kaddr3) : 0;
 	} else {
-		dev_err(client->dev, "rejected\n");
+		dev_warn(client->dev, "rejected, invalid param\n");
 		priv->status = -EINVAL;
 		priv->kbuf = NULL;
 		priv->size = 0;
-- 
2.18.1

